### DISCUSSION TITLE
Saturday 01/29/22 Meeting Agenda

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/omi-chan#2] [OMI-Chan Meetings]
OMI-chan Meeting 1 is happening on Saturday January 29th, 2022 from 20 to 21 UTC
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).

## Agenda

- Review what we have so far for OMI-Chan and the purpose of the group
  - Decide if we want to change that purpose any
- Decide on meeting cadence, including scheduled retrospectives (when you'll reflect on the last month and discuss what you will do differently moving forward - THIS IS CRITICAL, I recommend using this [quick working agreement template](https://github.com/omigroup/omi-chan/discussions/4) I have created in the omi-chan repo. 
  - Once you have your meeting cadence and times, just share with @mrmetaverse and I will get it added to the main omigroup readme :rocket:
- Review [readme fundamentals](https://github.com/omigroup/omi-chan), who we are, why we exist, and how to get involved 
- See OMI-Chan [Milestone 1](https://github.com/omigroup/omi-chan/milestone/1) to get comfortable using github issues.
  - As you complete the three issues in Milestone 1, you should notice the progress bar advance as you move issues to "done" on the [omi-chan kanban board](https://github.com/omigroup/omi-chan/projects/1)
- Decide how we want to accomplish the purpose of the group

**Comment by technobaboo** [comment]
Vote to make these 3 the purposes of this group:
1. To establish mascots, accessories, and apparel that embody OMI's core principles 
2. To create avatars for other community groups to promote interoperability 
3. Promote OMI in events and media using our mascots

In Favor:
@humbletim 
@fire 
@liftedpixel
@jeanahelver

Opposed:
nobody lol

**Comment by technobaboo** [comment]
Proposed meeting time is the same, 20-21 UTC on Saturdays. Unknown if everyone can attend, so subject to change.
